### ISSUE TITLE
Reproducing RAFT experiments (Table 3)

### ISSUE THREAD

**Issue by dgiova** [huggingface/setfit#147]
Hi, I wasn't able to locate the code to reproduce Table 3. I looked in the `scripts` folder but didn't have success.
Any help with this is greatly appreciated!

A side question on the RAFT results: did you use 10 random seeds for this experiment?

**Comment by MosheWasserb** [comment]
Hi, You will find all details on SetFit RAFT submission in my blog: 
https://towardsdatascience.com/sentence-transformer-fine-tuning-setfit-outperforms-gpt-3-on-few-shot-text-classification-while-d9a3788f0b4e.
Since test data is unavailable (RAFT is a blind submission) you will not be able to validate for accuracy. 
I recommend using the updated implementation from Hugging Face with the default hyperparameters (num_iteration=20, bs=16, epochs=1).

**Comment by tomaarsen** [comment]
I believe this has been answered now, so I'll close it.
